data/coalition/lunarium intro.txt

```diff
@@ -2050,3 +2050,80 @@ event "first lunarium unlock"
 		outfitter "Lunarium Basics"
 	planet "Refuge of Belugt"
 		outfitter "Lunarium Basics"
+
+
+
+# Patch for if the player joined both Heliarch and Lunarium prior to a fix.
+mission "Coalition Intro Join Patch"
+	landing
+	invisible
+	to offer
+		has "joined the lunarium"
+		has "joined the heliarchs"
+		has "event: first lunarium unlock"
+		has "event: first heliarch unlock"
+		has "license: Heliarch"
+	on offer
+		conversation
+			`(Currently, you have joined both the Coalition's Lunarium and Heliarch factions due to an unintended bug. Please choose which side you prefer to be with.)`
+			choice
+				`	(Side with the Heliarchs.)`
+					goto "heliarch"
+				`	(Side with the Lunarium.)`
+
+			action
+				clear "license: Heliarch"
+				clear "joined the heliarchs"
+				event "fix: chose heliarchs"
+				fail
+
+			label "heliarch"
+				clear "joined the lunarium"
+				event "fix: chose lunarium"
+				fail
```

Comment:
This straight up doesn't work. Also, these events look flipped.
```suggestion
			action
				clear "license: Heliarch"
				clear "joined the heliarchs"
				event "fix: chose lunarium"
			`	(You've sided with the Lunarium.)`
				decline

			label "heliarch"
			action
				clear "joined the lunarium"
				event "fix: chose heliarchs"
			`	(You've sided with the Heliarchs.)`
				decline
```

Reply:
Does it not work because it needs dialog or conversation?

Reply:
1. `label "heliarch"` didn't have an `action` below it, so the heliarch changes didn't work.
2. `action fail` doesn't just end the conversation, so if someone chose Lunarium, it'd run both actions.